apps/namehashlabs.org/components/2 - molecules/contact-us-section.tsx

```diff
@@ -28,7 +23,7 @@ export const ContactSection = () => {
           </p>
         </div>
         <div className="w-full md:w-auto">
-          <CalendarButton variant="secondary" className="!w-full">
```

Comment:
There are some screen widths where the layout here breaks. Perhaps it is related to "!w-full" being removed? I'm not familiar with what that does :)

![CleanShot 2024-10-21 at 17 11 58](https://github.com/user-attachments/assets/6dc99fed-3666-4ed7-82f4-c3d4f6253ae0)

Reply:
@lightwalker-eth done! :)